tests/checkpoint/conftest.py

```diff
@@ -5,8 +5,11 @@
 
 from great_expectations import DataContext
 from great_expectations.core import ExpectationConfiguration
+from great_expectations.core.yaml_handler import YAMLHandler
 from great_expectations.data_context.util import file_relative_path
 
+yaml = YAMLHandler()
```

Comment:
Do we need this on the global scope?

Reply:
Good call, no we only need it for the one fixture. I can move it down